lib/fathead/caniuse/parse.py

```diff
@@ -31,15 +31,25 @@ def generate_answers(data):
         ])
         print ','.join(titles)
 
+        # Generate Related Topics
+        links = feature_data.get('links', [])
+        related = u' '.join([u'[[{} {}]]'.format(link['title'], link['url']) for link in links])
+
         # Generate abstract
         abstract = feature_data['description']
```

Comment:
@samskeller let's also wrap the text description in its own `<p></p>` to separate it from the list of supported browsers. Right now it's one giant text blob that's hard to understand:

![flexible_box_layout_module__browser_support__at_duckduckgo](https://cloud.githubusercontent.com/assets/873785/17713012/bc87a4b4-63c6-11e6-9651-ddb1e6dd8d56.png)